nixos/modules/services/desktops/espanso.nix

```diff
@@ -0,0 +1,25 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let cfg = config.services.espanso;
+in {
+  meta = { maintainers = with stdenv.lib.maintainers; [ numkem ]; };
```

Comment:
`stdenv` isn't declared anywhere here, so you should just stick to `lib.maintainers`...